src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/CrossCaseTypeSearchRequest.java

```diff
@@ -43,13 +44,15 @@
     private final ElasticsearchRequest elasticsearchRequest;
     private final boolean multiCaseTypeSearch;
     private final List<String> aliasFields = new ArrayList<>();
+    private boolean consolidationQuery;
 
```

Comment:
If the response for all query types is going to be the same may be the attribute can be removed .   private boolean consolidationQuery;